components/components.html

```diff
@@ -95,8 +95,40 @@ <h2>Dot Badge</h2>
     </section>
     <hr>
     <section class="button">
-
+        <h1>Buttons</h1>
+        <div  class="button-container">
+            <p>Primary Buttons</p>
+            <div>
+                <button class="btn btn-primary btn-primary-dark">Button One</button>
+                <button class="btn btn-primary btn-primary-light">Button Two</button>
+                <button class="btn btn-primary">Button Three</button>
+            </div>
+        </div>
+        <div  class="button-container">
+            <p>Rounded Buttons</p>
+            <div>
+                <button class="btn btn-rounded btn-primary btn-primary-dark">Button One</button>
+                <button class="btn btn-rounded btn-primary btn-primary-light">Button Two</button>
+                <button class="btn btn-rounded btn-primary">Button Three</button>
+            </div>
+        </div>
+        <div  class="button-container">
+            <p>Buttons With Link</p>
+            <div>
+                <a href="#" class="btn btn-link">Google.com</a>
+                <a href="#" class="btn btn-link">Read More..</a>
+                <a href="#" class="btn btn-link">Go to next page</a>
+                
+            </div>
+        </div>
+        <iframe
+  src="https://carbon.now.sh/embed/ZjU3bqueLdKQikyfD1jM"
+  style="width: 930px; height: 564px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
     </section>
+    <hr>
+    </body>
```

Comment:
Remove this tag from here. Body tag is closed at end of file after all the sections